staging/src/k8s.io/noderesourcetopology-api/pkg/apis/topology/v1alpha1/types.go

```diff
@@ -0,0 +1,140 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	_ "github.com/gogo/protobuf/gogoproto"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// TopologyManagerPolicy represents kubelet configuration for
+// the TopologyManager component for a worker node.
+type TopologyManagerPolicy string
+
+const (
+	// SingleNUMANodeContainerLevel represent single-numa-node policy of
+	// the TopologyManager
+	SingleNUMANodeContainerLevel TopologyManagerPolicy = "SingleNUMANodeContainerLevel"
+	// SingleNUMANodePodLevel enables pod level resource counting, this policy assumes
+	// TopologyManager policy single-numa-node also was set on the node
+	SingleNUMANodePodLevel TopologyManagerPolicy = "SingleNUMANodePodLevel"
+	// Restricted TopologyManager policy was set on the node
+	Restricted TopologyManagerPolicy = "Restricted"
```

Comment:
`NodeResourceTopology` does not expose Topology Manager Scope and as of now it is mixed with policy name, we should add other policies for completeness: 
- `RestrictedPodLevel`
- `RestrictedContainerLevel`
- `BestEffortPodLevel`
- `BestEffortContainerLevel`

as per discussion in https://github.com/k8stopologyawareschedwg/noderesourcetopology-api/issues/21

Reply:
Added, Thanks!